pkgs/development/python-modules/pur/default.nix

```diff
@@ -1,19 +1,27 @@
 { lib
-, buildPythonPackage
-, click
+, python3
 , fetchFromGitHub
-, pytestCheckHook
 }:
 
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      self = py;
```

Comment:
```suggestion
```

don't think this line is necessary? at least, I see some other packages that don't do this